gruntfile.js

```diff
@@ -17,6 +17,9 @@ module.exports = function( grunt ) {
 
 		jshint: {
 			options: {
+				globals: {
+					'CKEDITOR': false
```

Comment:
> Keeping consistency makes searching properties initializations much easier - you just make ctrl + f on whole directory with <propName>: and you hit all the occurences.

The only thing is that you'll never work on such properties directly (like retrieving `globals.CKEDITOR`). You'll just work on the list as a whole, so quick access to the properties is usually useless.

Still I don't see a big deal for having it without quotes. I was just justifying why quotes ended up there. You guys may decide to remove them.